src/Site/Settings/Redirects.tsx

```diff
@@ -12,135 +13,144 @@ export const Redirects: FC<WhpptTab> = () => {
     { text: 'To', align: 'start', value: 'to' },
     { text: 'Published', align: 'start', value: 'published' },
     { text: 'Published At', align: 'start', value: 'publishedAt' },
-    { text: 'Last Modified', align: 'start', value: 'lastmod' },
+    { text: 'Last Modified', align: 'start', value: 'updatedAt' },
     { text: 'Created At', align: 'start', value: 'createdAt' },
   ] as any;
 
-  const [searchRedirects, setSearchRedirects] = useState('');
+  const { api, domain } = useWhppt();
+
+  const [errorState, setError] = useState<Error>();
+
   const [items, setItems] = useState([]);
+  const [total, setTotal] = useState(0);
+  const [searchRedirects, setSearchRedirects] = useState('');
 
   const [isAddingRedirect, setIsAddingRedirect] = useState(false);
+
   const [newRedirectName, setNewRedirectName] = useState('');
```

Comment:
Is it better to do these 3 use states (name, to, from) or to do one that is an object.